### PR TITLE
Add environment variable to enable mlir exhaustive tuning configs

### PR DESCRIPTION
**Pull request by pfultz2**

### CODE REVIEW

File: src/targets/gpu/mlir.cpp [hunk 1]
```diff
@@ -69,6 +69,7 @@ inline namespace MIGRAPHX_INLINE_NS {
 namespace gpu {
 
 MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_TRACE_MLIR);
+MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_MLIR_TUNE_EXHAUSTIVE);
```
Comment (umangyadav) [inline src/targets/gpu/mlir.cpp:72]: should we tie this to `exhaustive_tune` compile option as well ?

Comment (krzysz00) [reply src/targets/gpu/mlir.cpp:72]: I don't think so. The default of exhaustive tuning should be our "full", with full-on exhaustive being a debug feature

**Comment by bpickrel** [comment]
I think assigning this to me was a typo.

**Review by krzysz00 — APPROVED** [review]
LGTM

**Pull request merged**